File: net/yukisalon/yukisalon/ClientApp/src/app/models/Product.ts
export interface Product {
  id: number,
  name: string,
  description: string,
  price: string,
  isFixPrice: boolean,
  currency: string,
  image: string,
  isFeatured: string
}

export interface Category {
  id: number,
  name: string,
  image: string,
  product: Product[]
}

File: api/YukiSalonApi/YukiSalonApi/Models/Product.cs
using System;
using System.Collections.Generic;

namespace YukiSalonApi.Models
{
    public partial class Product
    {
        public int Id { get; set; }
        public int CategoryId { get; set; }
        public int? ImageId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public bool? IsFixPrice { get; set; }
        public string Currency { get; set; }
        public bool? IsFeatured { get; set; }
        public bool IsActive { get; set; }

        public virtual Category Category { get; set; }
        public virtual Image Image { get; set; }
    }
}
